app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -9,6 +9,7 @@ import Clan from '../../models/Clan'
 const JOB_TYPE = 'outcomes-report'
 
 const orgKinds = {
+  'administrative-country': { childKinds: ['school-district'] },
```

Comment:
I wonder if the `childKinds` here should be a bit smarter than this. Like, if it's in a country where we have configured administrative regions (just the US states for now), then use those. Otherwise, if it's in a country where we have configured school districts (just the US for now), then use those. But all the non-US countries both don't have configured school districts (from NCES) or administrative regions, so we should just go straight to school level?

Reply:
okay. added school-district for US now. let's add `school` and region level in next pr.